src/dom-to-image-more.js

```diff
@@ -298,7 +313,8 @@
         }
 
         function processClone(original, clone) {
-            if (!util.isElement(clone)) {
+            if (!util.isElement(clone)) { return Promise.resolve(clone); }
```

Comment:
This may fix something I randomly saw broken, hmmm

Reply:
The `Promise.resolve` was there before - I didn't see a rationale to kill it so took the cautious route and changed it back